client/docs/index.md

```diff
@@ -15,4 +15,3 @@ If you've read the documentation and still have questions, checkout our [main do
 
```

Comment:
The first line, `Featureform's`. Also, should we just call it SDK or API Reference?

Reply:
Also, is it possible to have landing pages for `Getting Started`, `Registration`...? Because when you click on it, it just drops down.